### PR TITLE
Added data-cy selectors

### PR DESCRIPTION
**Pull request by Cich0sza**

### CODE REVIEW

File: cogboard-webapp/src/components/AddBoard.js [hunk 1]
```diff
@@ -38,7 +38,8 @@ const AddBoard = () => {
     <>
       <IconButton
         onClick={handleAddBoardClick}
-        color="primary">
+        color="primary"
+        data-cy="add-board-add-button">
```
Comment (GrzegorzBednarski) [inline cogboard-webapp/src/components/AddBoard.js]: Shouldn't be board-add-button ?

File: cogboard-webapp/src/components/DropdownField.js [hunk 2]
```diff
@@ -12,7 +12,8 @@ const DropdownField = props => {
     name,
     children,
     dropdownItems,
-    itemsUrl
+    itemsUrl,
+    ...other
```
Comment (GrzegorzBednarski) [inline cogboard-webapp/src/components/DropdownField.js]: Is this change should be a part of this commit?

File: cogboard-webapp/src/components/DropdownField.js [hunk 3]
```diff
@@ -50,6 +51,7 @@ const DropdownField = props => {
         value={value}
         input={<Input name={name} id={id} />}
         name={name}
+        inputProps={other}
```
Comment (GrzegorzBednarski) [inline cogboard-webapp/src/components/DropdownField.js]: Is this change should be a part of this commit?

File: cogboard-webapp/src/components/UserLogin.js [hunk 4]
```diff
@@ -86,6 +88,7 @@ const UserLogin = () => {
             margin="normal"
             value={values.username}
             onKeyPress={(ev) => handleLoginOnEnterPress(ev, values)}
+            inputProps={{'data-cy': 'user-login-username-input'}}
```
Comment (GrzegorzBednarski) [inline cogboard-webapp/src/components/UserLogin.js]: Better will be login-form-username input (same situation with password, submit and validations in this form)

File: cogboard-webapp/src/components/AddBoard.js (hunk 1 above)
Comment (szymon-owczarzak) [reply cogboard-webapp/src/components/AddBoard.js]: Looks ok by me

File: cogboard-webapp/src/components/DropdownField.js (hunk 2 above)
Comment (szymon-owczarzak) [reply cogboard-webapp/src/components/DropdownField.js]: Yes - 

> and all the other props given

File: cogboard-webapp/src/components/NavBar.js [hunk 5]
```diff
@@ -44,6 +44,7 @@ const NavBar = ({ handleDrawerToggle }) => {
             aria-label="Open boards drawer"
             color="inherit"
             edge="start"
+            data-cy="navbar-boards-drawer-icon"
```
Comment (szymon-owczarzak) [inline cogboard-webapp/src/components/NavBar.js]: `navbar-show-drawer-button`

Comment (Cich0sza) [reply cogboard-webapp/src/components/NavBar.js]: done :)

**Pull request merged**